src/appShell/App/Container.tsx

```diff
@@ -90,6 +91,14 @@ export default class Container extends React.Component<IContainerProps, {}> {
 
                 <div className="pageTopContainer">
                     <UserMessager />
+                    {localStorage.getItem('warning_dismiss') !== 'Yes' &&
```

Comment:
that key should be more specific to this particular case.   'private_study_link_warning'